lib/components/SelectionControl/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./ToggleSwitch";
```

Comment:
according to @jia-deriv , we're following Pascal for the main folder and kebab for the internal files and folders. `./ToggleSwitch` -> `./toggle-switch` :)

Reply:
done , thanks